README.md

```diff
@@ -7,53 +7,126 @@ Forklift provides tools to create Foreman/Katello environments for development,
 * Virtualization enabled in BIOS
 
 See [Installing Vagrant](docs/vagrant.md) for installation instructions.
 
 ### Quickstart
 
-This will walk through the simplest path of spinning up a production test environment of a bleeding edge nightly installation assuming Vagrant and Libvirt are installed and configured.
+The quickstart guide covers the basic steps to install Forklift so that you can quickly spin up a production test environment of the latest bleeding edge nightly Foreman or Katello build.
 
-```
+It assumes you have already Vagrant and libvirt installed, as well as the vagrant-libvirt and vagrant-hostmanager plugins. You also need Ansible, although if you don't have it or aren't sure if your version is compatible, you can simply create and activate a Python virtual environment and then `pip install ansible-galaxy`
+
+Start by cloning the Forklift repository:
+
+```sh
 git clone https://github.com/theforeman/forklift.git
 cd forklift
+```
+
+Then install the Ansible collections which are needed by Forklift:
+
+```sh
 ansible-galaxy collection install -r requirements.yml
-vagrant up centos7-foreman-nightly
 ```
 
-The same can be quickly done for a development environment where GITHUB_NICK is your GitHub username:
+When this is complete, simply follow the next section of this README to try out the latest nightly Foreman build using Forklift.
+
+### Foreman Nightly Box
+
+Spin up your box and start using the latest nightly build of Foreman:
+
+```sh
+vagrant up centos8-foreman-nightly
+```
+
+Access the CLI by first connecting to the box via SSH:
 
+```sh
+vagrant ssh centos8-foreman-nightly
 ```
-git clone https://github.com/theforeman/forklift.git
-cd forklift
-cp vagrant/boxes.d/99-local.yaml.example vagrant/boxes.d/99-local.yaml
-sed -i.bak "s/<REPLACE ME>/GITHUB_NICK/g" vagrant/boxes.d/99-local.yaml
-vagrant up centos7-katello-devel
+
+To access the WebUI, it's helpful to have the vagrant-hostmanager plugin installed, so that your Workstation will automatically be able to resolve the hostname of the box to its IP address.
+
+Then you can simply open your browser and navigate to `https://centos8-foreman-nightly.<HOSTNAME>.example.com` where `<HOSTNAME>` is replaced by the shortname of your workstation. The first time you do this you will need to accept the self-signed certicate.
+
+By default, `forklift` deploys Foreman with `admin`/`changeme` as username and password.
+
+### Katello Nightly Box
+
+Katello nightly boxes are available as well; simply change `centos8-foreman-nightly` to `centos8-katello-nightly` and the steps are otherwise exactly the same as above.
+
+### Additional Documentation
+
+The remainder of this README contains helpful notes on additional topics that are likely to be of interest to users. For the most thorough and complete guides however, please refer to the [docs page](https://theforeman.github.io/forklift).
```

Comment:
@ekohl it occurred to me that you were responding to the deleted line below, and it would have been easy to miss that I didn't actually remove it, but rather give it a separate heading and moved it closer to the top.

That said, I could move this further to the very top.